features/flows/elastic/src/main/java/org/opennms/netmgt/flows/elastic/NodeDocument.java

```diff
@@ -43,6 +43,9 @@
     @SerializedName("node_id")
     private Integer nodeId;
 
+    @SerializedName("interface_id")
+    private Integer interfaceId;
+
```

Comment:
With a little mind-bending this still fits. It's the ID of the interface that was used to assign this flow to this node.

Reply:
Is `NodeDocument.interfaceId` meaningful and consistently used for all three usages in `FlowDocument`? I.e. for the members:

```
    @SerializedName("node_dst")
    private NodeDocument nodeDst;
    @SerializedName("node_exporter")
    private NodeDocument nodeExporter;
    @SerializedName("node_src")
    private NodeDocument nodeSrc;
```
In that case it may be justifiable to stick with that name.

Reply:
Yes, it is.